### DISCUSSION TITLE
Windows 10 not supported?

### DISCUSSION THREAD

**Discussion by jonwbstr** [soundshed/soundshed-app#60] [Q&A]
I tried to install on windows 10 and got an error. The error log opened a blank file, but I found a valid log in the same folder that says "os version not supported"

If windows 10 is not supported, would you add a minimum windows version to the readme and perhaps the download link that currently just says "windows"

**Comment by webprofusion-chrisc** [comment]
That could be related to recent updates to Electron version 22 but as far as I know that stopped support from Windows 7, not Windows 10

**Reply by jonwbstr** [reply]
I can share the log if you would like

**Reply by webprofusion-chrisc** [reply]
Yep that sounds be great. I assume your windows is 64-bit, our app has never worked on 32-bit.

**Comment by jonwbstr** [comment]
This is an old 32bit laptop, gotta love vague error messages 🤣
